pkg/skaffold/runner/runner_test.go

```diff
@@ -210,7 +232,7 @@ func TestNewForConfig(t *testing.T) {
 
 			testutil.CheckError(t, test.shouldErr, err)
 			if cfg != nil {
-				b, d := WithTimings(test.expectedBuilder, test.expectedDeployer)
+				b, _, d := WithTimings(test.expectedBuilder, test.expectedTester, test.expectedDeployer)
```

Comment:
Should we check the tester too?

Reply:
ah yeah, I did this originally just to get it to compile while I was writing other tests and forgot to fix this. I'll update